Source/EmbeddedBoundary/WarpXInitEB.cpp

```diff
@@ -66,7 +66,7 @@ namespace {
         }
 
     private:
-        HostDeviceParser<3> m_parser;
+        amrex::ParserExecutor<3> m_parser;
```

Comment:
Probably a good chance to describe the constant:
```suggestion
        amrex::ParserExecutor<3> m_parser; //! function parser with three arguments (x,y,z)
```